airbyte-webapp/src/components/JobItem/components/ResetStreamDetails.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import styled from "./ResetStreamDetails.module.scss";
+
+interface IProps {
+  names: string[];
+}
+
+export const ResetStreamsDetails: React.FC<IProps> = ({ names }) => (
```

Comment:
According to the designs we should only hide the "overflow" in this component if the specific log isn't expanded, i.e. I feel we should pass the `isOpen` from the parent component in here, and only apply the overflow/ellipsis logic if it's collapsed.

If it's expanded and multiline we'll need still a bit more spacing between the lines. I feel it might actually be easier having it a flexbox with a `gap` in this case, to achieve the styling, but feel free to solve this either way.

Reply:
fixed